app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
```

Comment:
Why will the result have more than one key? For a particular benchmark run undergoing this job there is only one key possible.